src/ui/wxWidgets/SafeCombinationSetupDlg.cpp

```diff
@@ -84,7 +84,9 @@ SafeCombinationSetupDlg::SafeCombinationSetupDlg(wxWindow *parent, wxWindowID id
 #ifndef NO_YUBI
   SetupMixin(FindWindow(ID_YUBIBTN), FindWindow(ID_YUBISTATUS));
   m_pollingTimer = new wxTimer(this, POLLING_TIMER_ID);
```

Comment:
Do we even need to initialize the timer if polling isn't enabled?

Reply:
Changed as suggested.